plugins/node/opentelemetry-instrumentation-cassandra/test/cassandra-driver.test.ts

```diff
@@ -76,6 +86,18 @@ function assertSingleSpan(name: string, query?: string, status?: SpanStatus) {
   assertSpan(span, name, query, status);
 }
 
+function assertAttributeInSpan(
+  name: string,
+  attributes?: { attributeName: string; attributeValue: string }[],
+  query?: string,
+  status?: SpanStatus
```

Comment:
The 2 only invocations of this function do not set `query` and `status`.
I suggest removing them